### PR TITLE
Prompt for password while setting biometrics on an empty wallet

### LINKED ISSUE #901: [Security] Bypass fingerprint lock for iOS and android Metamask apps
Issue
Able to login to wallet with fingerprint log without any proper authentication

For the Metamask android (v0.2.0 (13)) and iOS (v0.2.0 (225)) app, the only option for requiring authentication for the app is to set a fingerprint lock.
There is no feature to set the password for app authentication.
However, if you try to enter an empty password on the app login page, you can access the wallet without the need for the fingerprint

Reproduction steps
1. Enable fingerprint login
2. Terminate the metamask app
3. Start up metamask app and see the fingerprint login prompt
4. Press cancel to go in to the password login screen
5. With an empty password, press login
6. Verify that you have access to the wallet

### PR DESCRIPTION
**Pull request by brunobar79**
Fixed #901 

Added a new screen to set the password while turning on Biometrics on an empty wallet (without a password).

### CODE REVIEW

File: app/components/Views/ChoosePasswordSimple/index.js [hunk 1]
```diff
@@ -0,0 +1,400 @@
...
+	passwordStrengthLabel: {
+		height: 20,
+		marginLeft: 5,
+		marginTop: 10,
+		fontSize: 12,
+		color: colors.fontSecondary,
+		textAlign: 'left',
+		...fontStyles.normal
+	},
+	// eslint-disable-next-line react-native/no-unused-styles
```
Comment (estebanmino) [inline app/components/Views/ChoosePasswordSimple/index.js]: why did you add these three styles? seems like you're not using it

File: app/components/Views/ChoosePasswordSimple/index.js [hunk 2]
```diff
@@ -0,0 +1,400 @@
...
+	animateOutLabel = label => {
+		Animated.timing(label === 'new' ? this.state.labelsScaleNew : this.state.labelsScaleConfirm, {
+			toValue: 0.66,
+			duration: 200,
+			useNativeDriver: true,
+			isInteraction: false
+		}).start();
+	};
+
+	getPasswordStrengthWord() {
```
Comment (estebanmino) [inline app/components/Views/ChoosePasswordSimple/index.js]: not sure but i believe we have this in like 3 places now, if is the case maybe put this under utils?

File: app/components/Views/ChoosePasswordSimple/index.js [hunk 3]
```diff
@@ -0,0 +1,400 @@
...
+	};
+
+	render() {
+		const startX = 0;
+		const startY = 0;
+		const width = 100;
+		const height = 24;
+		const initialScale = 1;
+		const endX = 0;
+		const endY = 50;
```
Comment (estebanmino) [inline app/components/Views/ChoosePasswordSimple/index.js]: this consts are going to be set each time this view rerenders, maybe putting as const up at the beginning?

File: app/components/Views/ChoosePasswordSimple/index.js (hunk 2 above)
Comment (brunobar79) [reply app/components/Views/ChoosePasswordSimple/index.js]: This is kinda critical and I don't believe such a big refactor is a good idea ATM

**Review by estebanmino — APPROVED** [review]
code looks good, minor comments

File: app/components/Views/ChoosePasswordSimple/index.js (hunk 1 above)
Comment (brunobar79) [reply app/components/Views/ChoosePasswordSimple/index.js]: They are used dinamically and the linter doesn't recognize it.

**Review by ibrahimtaveras00 — APPROVED** [review]
Looks good in all scenarios on both iOS and Android, QA Passed 👍

**Pull request merged**